### PR TITLE
Update Pilot Things integration description

### PR DESCRIPTION
**Pull request by sylveon**
Drive-by: fix an English error in the ThingsBoard integration description

### CODE REVIEW

File: ui/src/views/applications/integrations/PilotThingsCard.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ class PilotThingsCard extends Component {
             Pilot Things
           </Typography>
           <Typography variant="body2" color="textSecondary" component="p">
-            The Pilot Things integration forward messages to a Pilot Things instance.
+            Get access to more than 500 sensor decoders and encoders with the Pilot Things integration.
```
Comment (brocaar) [inline ui/src/views/applications/integrations/PilotThingsCard.js:43]: The "Get access to more than 500 sensor decoders" is more a marketing slogan. Could we change this back to a more functional description? :)

Comment (sylveon) [reply ui/src/views/applications/integrations/PilotThingsCard.js:43]: Would "The Pilot Things integration brings more than 500 sensor payload data decoders and measurement-based flows." work better?

Comment (brocaar) [reply ui/src/views/applications/integrations/PilotThingsCard.js:43]: I would prefer to align it with the other integrations :-) In the end the integration forwards data to your platform, it is not the integration itself providing the decoders. 

> The InfluxDB integration writes events into an InfluxDB time-series database.

> The myDevices integration forwards events to the myDevices platform.

> The ThingsBoard integration forward events to a ThingsBoard.io instance.

(that should have been forward**s** ;))

Comment (sylveon) [reply ui/src/views/applications/integrations/PilotThingsCard.js]: I've reverted it to just fixing the English mistake.

**Pull request merged**